README.md

```diff
@@ -31,10 +31,10 @@ the node package manager `npm`. Then, from inside the `client-js` directory you
 global and project-level dependencies:
 
 ```
-$ npm install -g gulp typescript jspm bower
+$ npm install gulp typescript jspm bower
```

Comment:
Should not have to do this anymore. All packages we need should be included in `package.json` and will get installed when you do `npm install`.